language/cloud-client/v1beta2/snippets.py

```diff
@@ -175,17 +177,15 @@ def entity_sentiment_text(text):
 
 def entity_sentiment_file(gcs_uri):
     """Detects entity sentiment in a Google Cloud Storage file."""
-    language_client = language_service_client.LanguageServiceClient()
-    document = language_service_pb2.Document()
+    client = language_v1beta2.LanguageServiceClient()
 
-    document.gcs_content_uri = gcs_uri
-    document.type = enums.Document.Type.PLAIN_TEXT
+    document = types.Document(gcs_content_uri=gcs_uri, type='PLAIN_TEXT')
 
-    encoding = enums.EncodingType.UTF32
+    encoding = 'UTF32'
```

Comment:
removing encoding, since it is no longer needed:  https://github.com/googleapis/googleapis/pull/408